src/plugins/unified_doc_viewer/public/components/doc_viewer_table/table_cell_actions.tsx

```diff
@@ -145,57 +154,60 @@ export const TableActions = ({
     [closePopover]
   );
 
+  const panelItems: PanelItem[] = [
+    {
+      name: toggleColumnsLabel,
+      'aria-label': toggleColumnsAriaLabel,
+      'data-test-subj': `toggleColumnButton-${field}`,
+      icon: 'listAdd',
+      onClick: onClickAction(onToggleColumn.bind({}, field)),
+    },
+    {
+      name: pinnedLabel,
+      'aria-label': pinnedAriaLabel,
+      icon: pinnedIconType,
+      'data-test-subj': `togglePinFilterButton-${field}`,
+      onClick: onClickAction(togglePinned),
+    },
+  ];
+
+  if (onFilter) {
```

Comment:
Noticed that the order of items got changed. Can we keep it as before?

before
<img width="435" alt="Screenshot 2024-01-22 at 11 09 50" src="https://github.com/elastic/kibana/assets/1415710/9a20ab7f-33c5-46d5-a12a-62f6995da367">

after
<img width="390" alt="Screenshot 2024-01-22 at 11 10 12" src="https://github.com/elastic/kibana/assets/1415710/6ed05053-a7f4-4fac-ba1f-2eade40249a7">

Reply:
Thanx Julia for the review, it must be ok now https://github.com/elastic/kibana/pull/175087/commits/fc4b02ef780542c9e50abc73c8fe770826ce1fed